Modules/Utils/Sources/PocketCastsUtils/General/BuildEnvironment.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+/// Describes the type of build environment the app is running in.
+/// Use `BuildEnvironment.current` to get the environment for the current running build.
+public enum BuildEnvironment {
+    /// From Xcode, or another DEBUG build
+    case debug
+
+    /// A release build from TestFlight
+    case testFlight
+
+    /// A release build from the AppStore
+    case appStore
+
+    /// Returns the `BuildEnvironment` for the current build
+    public static var current: BuildEnvironment = .determineCurrentEnvironment
```

Comment:
I'm using `.determineCurrentEnvironment` so the `.current` value is only calculated once since the environment state shouldn't need to change at all.